### DISCUSSION TITLE
Octopus

### DISCUSSION THREAD

**Discussion by FarukhPi** [community/community#30141] [Other Feature Feedback, Questions, & Ideas]
Hey I'm new to python and I just finished the basics pretty much and I want to make a game but the thing is I can't find python arcade I try but I get confused. I'm young so please help me!

**Comment by airtower-luna** [comment]
Do you possibly mean this repository? https://github.com/pythonarcade/arcade

The fastest way to find a Python package by name is usually through the [Python Package Index (PyPI)](https://pypi.org/), where I found this: https://pypi.org/project/arcade/

Good luck with the snake! :smile_cat: :snake:

**Comment by melvincwng** [comment]
@FarukhPi, check out the following resources on how to utilize the python arcade library for making games:

1. https://api.arcade.academy/en/latest/ (Official documentation)
2. https://opensource.com/article/18/4/easy-2d-game-creation-python-and-arcade (Technical blog)
3. YouTube has also some good resources on Python Arcade Library

Good luck in your video game making journey 🎮!
